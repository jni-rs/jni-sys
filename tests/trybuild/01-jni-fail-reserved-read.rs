
use jni_sys::JNINativeInterface_;

pub fn main() {
    unsafe {
        let jni = std::mem::zeroed::<JNINativeInterface_>();
        let _reserved = jni.v1_1.reserved0;
    }
}
