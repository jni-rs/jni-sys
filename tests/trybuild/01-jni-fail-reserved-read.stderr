error[E0609]: no field `reserved0` on type `JNINativeInterface__1_1`
 --> tests/trybuild/01-jni-fail-reserved-read.rs:7:34
  |
7 |         let _reserved = jni.v1_1.reserved0;
  |                                  ^^^^^^^^^ unknown field
  |
  = note: available fields are: `GetVersion`, `DefineClass`, `FindClass`, `GetSuperclass`, `IsAssignableFrom` ... and 203 others
